about.html

```diff
@@ -1,73 +1,86 @@
+                <a href="#products">Products</a>
+                <a href="#blog">Blog</a>
+                <a href="#contact">Contact</a>
+            </nav>
+        </header>
+        <div class="top-content">
+            <img class="image" src="C:\Users\coino\User-Interface\Sprint-Challenge--User-Interface\img\jumbo-about.png">
+            <div class="about-text-container">
+                <h2>About</h2>
+                <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view
+                    of disruptive innovation via workplace diversity and empowerment.</p>
+            </div>
+        </div>
+        <section class="top-content">
+            <div class="text-container">
+                <img src="img/about-plan.png" alt="strategy">
+                <h2>Strategy</h2>
+                <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+                <button><strong>Learn More</strong></button>
+            </div>
+
+            <div class="text-container">
+                <img src="img/about-working.png" alt="strategy">
+                <h2>How We Work</h2>
+                <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+                <button><strong>Learn More</strong></button>
+            </div>
+        </section>
+        <section class="top-content">
+            <div class="text-container">
+                <img src="img/about-office.png" alt="strategy">
+                <h2>Places We Work</h2>
+                <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+                <button><strong>Learn More</strong></button>
+            </div>
+            <div class="text-container">
+                <img src="img/about-meeting.png" alt="strategy">
+                <h2>Collaboration</h2>
+                <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+                <button><strong>Learn More</strong></button>
+            </div>
+
+        </section>
+
+        <div class="footer-content">
+            <h2>Let's Work Together</h2>
+            <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of
+                disruptive innovation via workplace diversity and empowerment.</p>
+        </div>
+
+
+        <footer>
+            <nav>
+                <a href="index.html">Home</a>
```

Comment:
This is the correct way to route/link another page. Just like you did it on this footer nav is how you should do it on the main nav above.